### ISSUE TITLE
Export startovky pro automaticke nacitani jinymi programy

### ISSUE THREAD

**Issue by xkenia** [fvacek/quickbox#343]
Pripravit export startovky, aby ji mohli vyuzivat dalsi programy, napr. EmmaClient.

Format bych zvolil
 1044  2058512 D10C    DOR0952 Baštová Justýna        246.00,0000
id zavodnika = int(5)
mezera
si = int(8)
mezera
kategorie = str(7)
mezera
registracka = str(7)
mezera
prijmeni jmeno = str(22)
mezera
startovni cas = mmmm.ss,zzzz

Startovku generovat bud rucne nebo co 5 minut.

**Comment by fvacek** [comment]
Mel bych k tomu 2 veci:
1. format casu bych videl hezci jako 246:58.999, nevim, jestli je potreba merit na 4 des. mista, QE to ani nepodporuje, to je kosmeticka zalezitost
1. Formaty s pevnou sirkou sloupcu jsou dobre pro lidi ale pro programatory je to nejhorsi opruz, blbe se to generuje i parsuje, to nemluvim o tom, ze zavodnik s delsim jmenem o kus prijde. Takze se primlouvam za CSV format, pokud ovsem napr. EmmaClient CSV umi

**Comment by xkenia** [comment]
Ano, formaty nejsou nejlepe zvolene, ale duvod je ten, ze uz je to v Racomu a v EmmaClientu takto implementovano. EmmaClient CSV umi, ale nevim o tom, ze by umel variantu csv a Racom.

**Comment by jmacura** [comment]
nějaký racionální důvod, proč nepoužít formát IOF XML 3.0?

**Comment by fvacek** [comment]
EmmaClient CSV export je jiz napsan, takze ho mazat nebudeme, navic pochybuji o tom, ze by Racom umel IOF XML 3.0, takze se budeme muset prizpusobit my, ale to nevadi. 

QE z hidtorickych duvodu podporuje export vysledku do IOF XML 2.3, rozhodne by bylo fajn, prejit na 3.0 nebo ho alespon dopsat.

**Comment by arnost00** [comment]
1) Startovka - to jsem si myslel ze XML3.0 je uz udelana a pouziva se pro ORIS?
2) Vysledky / mezicasy - podporu XML 3.0 beru, ale prijde mi to hrozne neefektivni pro online mezicasy pouzivat format dat co ma nejvetsi pomer velikosti dat / prenesene informace. A k tomu se pri kazdem novem mezicasu zapise a precte i cela startovka znovu. 

Historie - napojeni racomu na Emma client se resilo v dobe pred QE a vychazelo se z exitujicich formatu pouzivanych pro mezicasy od Racomu (mezicasy, startovky).
